AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -747,8 +810,8 @@ def populate_asset_info(workspace, project, assets):
                                            + product.product_name
 
     @staticmethod
-    def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_path):
-        debug_graph_path = os.path.join(asset_processor.project_test_cache_folder(), actual_file_path)
+    def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_path, expected_hashes_to_skip = None, actual_hashes_to_skip = None):
```

Comment:
nit: Prefer to provide type hints in python methods.

Reply:
Done, added